test/integration/operator_policy_complianceconfig_test.go

```diff
@@ -0,0 +1,208 @@
+				1,
+			).Should(Equal(policiesv1.NonCompliant))
+			By("Checking the status messages of the operator policy")
+			Eventually(
+				common.GetOpPolicyCompMsg(operatorPolicyName),
+				defaultTimeoutSeconds,
+				1,
+			).Should(ContainSubstring("InstallPlan to update to [" + operatorVersionRecent +
+				"] is available for approval"))
+		})
+
+		It(operatorPolicyName+" should be patched with Automatic upgradeApproval", func() {
+			_, err := common.OcHub(
+				"patch", "policies.policy.open-cluster-management.io", policyName,
+				"-n", userNamespace, "--type=json", "-p", `[{
+				"op":"replace", 
+				"path":"/spec/policy-templates/0/objectDefinition/spec/upgradeApproval",
+				"value":"Automatic"
+			}]`)
+			Expect(err).ToNot(HaveOccurred())
+		})
+
+		It(operatorPolicyName+" should become Compliant after modifications to upgradeApproval", func() {
+			By("Checking if the status of the operator policy is Compliant")
+			Eventually(
+				common.GetComplianceState(policyName),
+				defaultTimeoutSeconds,
+				1,
+			).Should(Equal(policiesv1.Compliant))
+			Eventually(
+				common.GetOpPolicyCompMsg(operatorPolicyName),
+				defaultTimeoutSeconds,
+				1,
+			).Should(ContainSubstring("ClusterServiceVersion (" + operatorVersionRecent +
+				") - install strategy completed with no errors"))
+		})
+
+		// RHACM4K-48382 - Install operator via set all options for spec.complianceConfig as NonCompliant in the policy
+
+		It(operatorPolicyName1+" should be Compliant and OperatorGroup and Subscription should be created", func() {
+			By("Checking if the status of the operator policy is Compliant")
+			Eventually(
+				common.GetComplianceState(policyName1),
+				defaultTimeoutSeconds,
+				1,
+			).Should(Equal(policiesv1.Compliant))
+			By("Checking the status messages of the operator policy")
+			Eventually(
+				common.GetOpPolicyCompMsg(operatorPolicyName1),
+				defaultTimeoutSeconds,
+				1,
+			).Should(SatisfyAll(
+				ContainSubstring("the OperatorGroup matches what is required by the policy"),
+				ContainSubstring("the Subscription matches what is required by the policy"),
+				ContainSubstring("all operator Deployments have their minimum availability")))
+		})
+
+		It(operatorPolicyName1+"should have installed the intended operator "+operatorVersionInitial, func() {
+			By("Checking status of the operator policy")
+			Eventually(
+				common.GetOpPolicyCompMsg(operatorPolicyName1),
+				defaultTimeoutSeconds,
+				1,
+			).Should(ContainSubstring("ClusterServiceVersion (" + operatorVersionRecent +
+				") - install strategy completed with no errors"))
+		})
+
+		It(operatorPolicyName+" should be patched with NonCompliant ComplianceConfig for each option", func() {
+			// new policy specifies NonCompliant for each field of complianceConfig
+			_, err := common.OcHub("apply", "-f", policyPath1Modified, "-n", userNamespace)
+			Expect(err).ToNot(HaveOccurred())
+		})
+
+		It(operatorPolicyName1+" should remain Compliant", func() {
```

Comment:
This `It` should be combined with the one above it since they are part of the same test.